src/components/map/layers/edit/EditNodeLayer.tsx

```diff
@@ -68,4 +70,4 @@ class EditNetworkLayer extends Component<IEditNetworkLayerProps> {
     }
 }
 
-export default EditNetworkLayer;
+export default withLeaflet(EditNodeLayer);
```

Comment:
withLeaflet is not needed, since we dont use leaflet in this class

Reply:
Fixed